Update casing of IMUTracker to version 2

y6tada opened the pull request:
I designed new casing for IMUTracker with Autodesk Inventor. Also full assemble cad model of QUIKS and other parts for assembling are included in those commits.
In this update, connector direction issue that previous casing "Housing.SLDPRT" have had fixed, and the previous one renamed with "IMUTracker_casing_v1.SLDPRT".
Also, I attached appearance image to Hardware/ and displayed it on README, hope you like it though...🤔

Commit:
Update .gitignore as a preparation of adding autodesk inventor's file

Commit:
Create IMUTracker_casing_v2.ipt

Commit:
Rename Housing.SLDPRT --> IMUTracker_casing_v1.SLDPRT, add full assemble of QUIKS

Commit:
change location of image

Commit:
Add appearance image

Commit:
Move image to bottom side

Commit:
add STL file for 3d printing, accordingly add & change directoriy

mtkrtk commented:
I have some request

1. Change directory structure
```
Hardware - IMUTracker
         └ IMUTracker_casing
```
into
```
Hardware - IMUTracker - Circuit
                      └ Casing
```

2. Upload IMUTracker_assemble.png to gist
3. Create new README.md at Hardware/IMUTracker
4. Refer IMUTracker_assemble.png from Hardware/IMUTracker/README.md
5. Update .gitattribute file

Commit:
Changed directory structure, remove appearance image

Commit:
Update IMUTracker_assemble.iam according to the change of directory structure, add README into Hardware/IMUTracker, uplpad step cad model of full assemble.

Commit:
Merge remote-tracking branch 'upstream/master'

Commit:
Update .gitattributes

ipt & iam are Inventor's file format. stl is 3d print data format, and step is a exchange format.

Commit:
Fix image url in gist

y6tada commented:
According to your requests,

1. I changed directory str, also I added "IMUTracker_assemble.stp" into Hardware/IMUTracker/Casing/STEP

1.

y6tada commented:
According to your requests,

1. I changed directory str, also I added "IMUTracker_assemble.stp" into Hardware/IMUTracker/Casing/STEP
2. Remove appearance image form TOP README, and remove .png from Hardware/
Then I uploaded image into my gist [here](https://gist.github.com/y6tada/de85adfb4af9ef8d579c89646622f2fe#file-image
)
3. Created new README at Hardware/IMUTracker
Do I need to attach some specific license for Hardware resources? If my understanding is correct, my resources are currently under the license of this entire project (Apatch). If so, that is totally no problem for me.
4. I did displayed images on Hardware/README
5. Updated .gitattribute
Then, I actually modified previous attribute you wrote, please confirm how it works.

Regards,

komori-t reviewed (changes requested):
Some files (like .SLDPRT) are not specified as binary.
Specifying directories is easier.
Also, put LF on EOF.

Commit:
Specified IMUTracker/Casing/ directory as binary. Update step cad file (maybe automatic backup save with clising this file last time)

y6tada commented:
Now I specified "Hardware/IMMITracker/Casing/**" as binary.
Also added LF at the end line of .gitattributes

komori-t commented on .gitattributes:
```diff
@@ -1,4 +1,2 @@
-Hardware/IMUTracker_casing/**/* binary
-Hardware/IMUTracker/IMUTracker.step binary
-
+"Hardware/IMUTracker/Casing/**" binary
```
correct wildcard usage is:
`Hardware/IMUTracker/Casing/**/*`

Commit:
Fix typo (?) QUICKS --> QUIKS on top README

Commit:
Fixed wildcard usage on .gitattributes

Commit:
add LF at the end of top README

komori-t reviewed (approved):
LGTM
